source/_docs/guides/front-end-performance.md

```diff
@@ -0,0 +1,457 @@
+
+  <!-- 2nd Tab Nav -->
+  <li id="tab-2-id" role="presentation"><a href="#tab-2-anchor2" aria-controls="tab-2-anchor2" role="tab" data-toggle="tab">Drupal</a></li>
+</ul>
+
+<!-- Tab panes -->
+<div class="tab-content">
+  <!-- Active pane content -->
+  <div role="tabpanel" class="tab-pane active" id="tab-1-anchor2" markdown="1">
+Use the [Autoptimize](https://wordpress.org/plugins/autoptimize/){.external} plugin and the following configuration in your site's `wp-config.php` file:
+
+
+```php
+// Configure directory & filename of cached autoptimize files
+define('AUTOPTIMIZE_CACHE_CHILD_DIR','/uploads/resources/');
+define('AUTOPTIMIZE_CACHEFILE_PREFIX','aggregated_');
+```
+
+The following blog post walks through how to use results from the [Critical Path CSS Generator](https://jonassebastianohlsson.com/criticalpathcssgenerator/){.external} with the Autoptimize plugin so that you have inline critical styles with a deferred stylesheet: [How to Use the Autoptimize “Inline and Defer CSS” Option](https://www.wpfaster.org/blog/how-to-use-autoptimize-inline-and-defer-css-option){.external}
+  </div>
+
+  <!-- 2nd pane content -->
+  <div role="tabpanel" class="tab-pane" id="tab-2-anchor2" markdown="1">
+Drupal Core has the ability to "aggregate" CSS and JavaScript. When turned on, Drupal will combine individual CSS and Javascript files in a smaller number of bigger files. This easy optimization can be done at `admin/config/development/performance`:
+
+![Drupal performance admin interface](/source/docs/assets/images/guides/front-end-performance/drops-performance.png)
+
+Checking these boxes will take a normal Drupal site from having dozens (or hundreds) of small CSS/JS files to just a few. As you browse around the site the aggregated files loaded will be different as different modules add different source files to the page. [For more details, see the Drupal.org documentation on CSS organization.](https://www.drupal.org/docs/develop/standards/css/css-file-organization-for-drupal-8){.external}
+
+If you want to make the number of files as low as possible, try [Advanced Aggregation](https://www.drupal.org/project/advagg){.external} module. [It can help take fine grain control over how files are combined](http://www.mediacurrent.com/blog/better-css-js-aggregation-advanced-aggregation){.external}. In the past there have been bugs when using Advanced Aggregation on Pantheon. To our knowledge, those are all resolved.
+  </div>
+</div>
+
+<div markdown="1" class="alert alert-info">
+  <h4 class="info">Note</h4>
+The Pantheon Global CDN includes HTTP/2. One of the benefits of HTTP/2 is that it allows multiple files to be downloaded simultaneously.<br/><br/>
+The Drupal community has long speculated whether HTTP/2 will make CSS and JavaScript aggregation irrelevant. (In theory, lots of little files, each individually cacheable, and downloaded en masse over one HTTP/2 connection would be more performant than aggregated files.)<br/><br/>
```

Comment:
This is a bigger speculation than the Drupal community. I'd say "the web community"

Reply:
Also, the key tradeoff is whether you have one aggregated file (which can only be beaten by one big inline CSS statement) vs different pages serving different aggregates. But I digress. We can revisit this if/when we get into HTTP/2 Push support. ;)